src/ripple/protocol/impl/Feature.cpp

```diff
@@ -465,6 +465,7 @@ REGISTER_FIX    (fixAMMOverflowOffer,           Supported::yes, VoteBehavior::De
 REGISTER_FEATURE(PriceOracle,                   Supported::yes, VoteBehavior::DefaultNo);
 REGISTER_FIX    (fixEmptyDID,                   Supported::yes, VoteBehavior::DefaultNo);
 REGISTER_FIX    (fixXChainRewardRounding,       Supported::yes, VoteBehavior::DefaultNo);
+REGISTER_FIX    (fixPreviousTxnID,              Supported::yes, VoteBehavior::DefaultNo);
```

Comment:
if changes the binary protocol and the hashes; so if validators are out of sync with regard to this feature, they will lose consensus

Reply:
I think of it as a fix because IMO these fields should have been included on the objects, but they were "forgotten" (or maybe not deemed important enough, idk). This isn't a strong opinion though, fine with changing if you want me to.

Reply:
@bronek I should have been more clear - I was thinking that it would be a feature amendment instead of a fix amendment. Still an amendment, though.

@mvadari Yes, that reasoning makes sense. You've convinced me. Keep it as a fix.